pkg/kev/override.go

```diff
@@ -311,9 +387,16 @@ func (o *composeOverride) mergeServicesInto(p *ComposeProject) error {
 		if err != nil {
 			return err
 		}
-
 		envVarsFromNilToBlankInService(base)
 
+		// Copy over anything that is defined outside an extension or
+		// environment - users may modify some elements of the generated
+		// override files such as image names (for example, to provide
+		// different images for staging and prod environments)
+		if override.Image != base.Image {
```

Comment:
I think this works for now. Would it make sense to introduce an enum of things we want to respect or is it too early?

Reply:
Agree, fine for now, but will adjust if/when we start adding other fields to the 'precedence' list